deployment/deploy.yml

```diff
@@ -67,6 +67,13 @@ objects:
             initialDelaySeconds: 30
             periodSeconds: 5
             timeoutSeconds: 2
+          env:
```

Comment:
@SteveHNH I haven't added the environment variable overrides to app-interface, but is it looking right on this end?

Reply:
Currently clowder environment-variables aren't supported for front-ends. This is being worked on as part of [RHCLOUD-21050](https://issues.redhat.com/browse/RHCLOUD-21050). In the meantime, I'm adding an additional API endpoint to handle this configuration [through this PR](https://github.com/RedHatInsights/payload-tracker-go/pull/115).